oggm/utils.py

```diff
@@ -866,9 +866,9 @@ def srtm_zone(lon_ex, lat_ex):
 
     # quick n dirty solution to be sure that we will cover the whole range
     mi, ma = np.min(lon_ex), np.max(lon_ex)
-    lon_ex = np.linspace(mi, ma, np.ceil((ma - mi) + 3))
+    lon_ex = np.linspace(mi, ma, int(np.ceil((ma - mi) + 3)))  # int() to avoid Deprec warning
```

Comment:
Is this line (and others below) shorter than 79 characters? We try to apply PEP8 as far as possible